microservice-transaction-sample/docs/README.md

```diff
@@ -0,0 +1,443 @@
+...
+```
+
+These histories are ordered by timestamp in a descending manner.
+
+The current `credit_total` is `10000`, so it has reached the `credit_limit`.
+So, the customer can't place an order anymore due to the limit.
+
+```shell
+$ ./gradlew :client:run --args="GetCustomerInfo 1"
+...
+{"id": 1,"name": "Yamada Taro","credit_limit": 10000,"credit_total": 10000}
+...
+$ ./gradlew :client:run --args="PlaceOrder 1 3:1,4:1"
+...
+io.grpc.StatusRuntimeException: FAILED_PRECONDITION: Credit limit exceeded
+        at io.grpc.stub.ClientCalls.toStatusRuntimeException(ClientCalls.java:262)
+        at io.grpc.stub.ClientCalls.getUnchecked(ClientCalls.java:243)
+        at io.grpc.stub.ClientCalls.blockingUnaryCall(ClientCalls.java:156)
+        at sample.rpc.OrderServiceGrpc$OrderServiceBlockingStub.placeOrder(OrderServiceGrpc.java:295)
+        at sample.client.command.PlaceOrderCommand.call(PlaceOrderCommand.java:38)
+        at sample.client.command.PlaceOrderCommand.call(PlaceOrderCommand.java:12)
+        at picocli.CommandLine.executeUserObject(CommandLine.java:1783)
+        at picocli.CommandLine.access$900(CommandLine.java:145)
+        at picocli.CommandLine$RunLast.handle(CommandLine.java:2141)
+        at picocli.CommandLine$RunLast.handle(CommandLine.java:2108)
+        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:1975)
+        at picocli.CommandLine.execute(CommandLine.java:1904)
+        at sample.client.Client.main(Client.java:39)
+...
+```
+
+After repayment, the customer will be able to place an order again!
+
+```shell
+$ ./gradlew :client:run --args="Repayment 1 8000"
+...
+$ ./gradlew :client:run --args="GetCustomerInfo 1"
+...
+{"id": 1,"name": "Yamada Taro","credit_limit": 10000,"credit_total": 2000}
+...
+$ ./gradlew :client:run --args="PlaceOrder 1 3:1,4:1"
+...
+{"order_id": "dd53dd9d-aaf4-41db-84b2-56951fed6425"}
+...
+```
+
+## Clean up
+
+To stop Cassandra, MySQL and the Microservices, run the following command:
+```shell
+$ docker-compose down
+```
+
+## How the microservice transaction is achieved
+
+So far, you have run the sample application, but you haven't seen how it is implemented.
+Let's look at the code to see how the transaction that spans the services is implemented.
+
+Transaction #2 (Placing an order) achieves the microservice transaction, so we focus on this transaction here.
+
+The sequence diagram of transaction #2 is as follows:
+
+<p align="center">
+<img src="images/sequence_diagram.png" width="400" />
+</p>
+
+### 1. Start a two-phase commit transaction
+
+When a client sends `Place an order` request to Order Service, [OrderService.placeOrder()](../order-service/src/main/java/sample/order/OrderService.java#L102-L103) is called, and the microservice transaction starts.
```

Comment:
@josh-wong The source code files of the example application will also be copied to the doc site?

Reply:
Unfortunately, making source code files in static site generators can be a bit tough, even if a plugin is available.

As mentioned in [another comment](https://github.com/scalar-labs/scalardb-samples/pull/26#issuecomment-1475767352), I'll discuss and show a possible solution in our upcoming meeting.